controllers/chaosimpl/jvmchaos/impl.go

```diff
@@ -123,7 +123,9 @@ func (impl *Impl) Recover(ctx context.Context, index int, records []*v1alpha1.Re
 		}()
 	}
 	if err != nil {
-		return v1alpha1.Injected, err
+		// Unable to find the container, so we are unable to remove the experiment from the jvm as it has gone
+		impl.Log.Error(err, "finding container")
+		return v1alpha1.NotInjected, nil
```

Comment:
How about checking the error message? Only when the error message contains something means contained not found return `v1alpha1.NotInjected`

Reply:
Great! Maybe it's better to put the check under L131.